app/views/wpcc/your_details/new.html.erb

```diff
@@ -42,10 +39,7 @@
         <div class="details__field">
           <div class="details__field-label">
             <%= f.label :gender, t('wpcc.details.gender.label'), class: 'form__label-heading' %>
-            <button type="button" class="popup-tip__button" data-dough-popup-trigger>
-              <span aria-hidden="true">i</span>
-              <span class="visually-hidden"><%= t('wpcc.details.tooltip_show') %></span>
-            </button>
+            <%= render 'wpcc/shared/popup_tip_trigger' %>
```

Comment:
Sorry, but I am trying to avoid as much as possible to use a "shared" folder. The reason is this folder becomes the black hole of the project with a lot of views for different action and could become not so well organised (as it is on the frontend repo atm).
That's the reason I suggested "tooltip/show" to think more like the views in a resouce way.
Or if you really want to continue to use some sort of "shared" folder I will reject this option and move the views to app/views/wppc/engine/ folder in order to use template inheritance (more details you can see [here](http://guides.rubyonrails.org/layouts_and_rendering.html#template-inheritance)).